services/search-api/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "search-api",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "node ../../node_modules/.bin/jest"
+  },
+  "keywords": [],
+  "author": "Helsingborg Stad",
+  "license": "MIT",
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "^7.15.5",
+    "@babel/plugin-transform-modules-commonjs": "^7.15.4",
+    "babel-jest": "^27.2.0",
+    "jest": "^27.2.0",
+    "jest-environment-jsdom-sixteen": "^2.0.0"
+  }
```

Comment:
Maybe this should live in the package.json at root-level? We try to put global packages in here for other services to use.

Reply:
Yes, these dependencies are actually not needed in the service itself. They do exists in the root package.json

Reply:
Even better :)

Reply:
Removed dependencies and ran the tests again, everything works fine.